src/wallet.js

```diff
@@ -1637,6 +1637,7 @@ Wallet.prototype.createAndSignTransaction = function(params, callback) {
     const estimatedSize = transaction.estimatedSize;
```

Comment:
this gets calculated in transactionBuilder.js by counting the number of inputs/outputs and using those config constants we looked at yesterday. I ran a test on that function and it was returning a lower amount that tx.virtualSize(). tx.virtualSize seems to be exactly what ends up in the blockchain, so I think i'd rather stick with that than use this old variable/function

Reply:
Also this is the estimate before the fullSigned tx comes back from the platform. So i think its better to use the actual size calculated from the fullsigned tx rather than estimating based on the half-signed

Reply:
If it's the size of the half-signed, it should not be called `estimatedSize` that's misleading. The size of the half-signed should be called `partiallySignedSize` or similar. If it would make sense to have a decent estimate for the actual virtual size in this spot, we should be able to calculate that easily from the unsigned or partially signed tx, though.

Reply:
i think estimatedSize is estimating how big the tx will be once it is full-signed, by using predefined constants and counting the number of inputs/outputs